src/pages/Landing/components/Body/ChatBox/Input/Input.module.scss

```diff
@@ -0,0 +1,29 @@
+.Input {
+    padding: 1.6rem 1.3rem;
+    height: 7.2rem;
+    background: #ebe7f3;
```

Comment:
use variable

Reply:
Which value should you write in a variable?

Reply:
` background: #ebe7f3;` we should be standazing colors and should add in centralized variables to avoid different pallets getting used by other developers

Reply:
I can create $input-bg-color: #ebe7f3; variable in "valiables.scss" in the branch.

Reply:
added

Reply:
make changes for below 2 comments and push the code